### DISCUSSION TITLE
This ansible install is a joke... you loose hours to try and nothing works out

### DISCUSSION THREAD

**Discussion by leopucci** [openwisp/ansible-openwisp2#497] [Ideas]
![image](https://github.com/user-attachments/assets/6e1e9ca9-d6da-4be9-832f-898d160b9efd)

Hi, i think that this project can go above and beyond if you change the way that you install this application. 
If you want to keep offering this crapy install method, the least you can do is offer pre-installed virtual machines for clients. 
Most people these days install on    proxmox or vmware esxi... if you offer a template for virtual machines, you would help many people

**Comment by nemesifier** [comment]
Ranting is not a great way to get support from us. I am closing this and banning you, your attitude is not welcome here. Thanks.
